src/ui/Analytics/Analytics.ts

```diff
@@ -50,9 +50,6 @@ export interface IAnalyticsOptions {
  * You can use analytics data to evaluate how users are interacting with your search interface, improve relevance and
  * produce analytics dashboards within the Coveo Cloud Platform.
  *
- * See [Step 7 - Usage Analytics](https://docs.coveo.com/en/347/) of the Getting Started with the JavaScript
```

Comment:
Leads to this page which requires login https://levelup.coveo.com/learn/courses/jsui/lessons/javascript-search-framework-tutorial-6-usage-analytics